### PR TITLE
Hanlde internal load balancers

### LINKED ISSUE #10: doesn't work with internal ELB
internal ELBs' DNS is prepended by `internal-`, [this line](https://github.com/wearemolecule/route53-kubernetes/blob/master/service_listener.go#L172), that should retrieve the ELB name, doesn't work with that naming schema, because it returns `internal` as the dns name

### PR DESCRIPTION
**Pull request by itajaja**
close #10 

Please note that this is the first time I have ever written go code and that I did it inside the github editor :D

### CODE REVIEW

File: service_listener.go [hunk 1]
```diff
@@ -182,7 +182,14 @@ func serviceHostname(service *api.Service) (string, error) {
 }
 
 func hostedZoneId(elbApi *elb.ELB, hostname string) (string, error) {
-	elbName := strings.Split(hostname, "-")[0]
+	hostnameSegments := strings.Split(hostname, "-")
+	elbName := hostnameSegments[0]
+	
+	// handle internal load balancer naming
+	if elbName == 'internal' {
+		elbName := hostnameSegments[1]
```
Comment (stephenmhoc) [inline service_listener.go]: First off thanks for the PR! 

That being said, I think this line is incorrect. In golang the `:=` operator is used to instantiate new variables so the scope of this variable lives inside the `if` statement. It seems you just want to reassign the variable so in that case you just need the `=` operator.

Comment (itajaja) [reply service_listener.go]: TIL :)

**Comment by itajaja** [comment]
you might want to test it before merging it though :) (hound seems stuck btw)

**Comment by iterion** [comment]
Yep, testing it now :)

File: service_listener.go [hunk 2]
```diff
@@ -182,7 +182,14 @@ func serviceHostname(service *api.Service) (string, error) {
 }
 
 func hostedZoneId(elbApi *elb.ELB, hostname string) (string, error) {
-	elbName := strings.Split(hostname, "-")[0]
+	hostnameSegments := strings.Split(hostname, "-")
+	elbName := hostnameSegments[0]
+	
+	// handle internal load balancer naming
+	if elbName == 'internal' {
```
Comment (iterion) [inline service_listener.go]: Another small issue, `'internal'` should be `"internal"`. Double quotes are for strings and single quotes are for runes (aka chars).

**Comment by iterion** [comment]
I've tested this in our cluster and it still works for our use case. I'm going to go ahead and merge as the solution seems correct. I'm also going to add a test for this and create a new tag. Would someone from #10 mind testing this. I'll comment there when a new tag is available.

**Pull request merged**